src/jit/codegenxarch.cpp

```diff
@@ -8381,6 +8381,14 @@ void CodeGen::genCreateAndStoreGCInfoX64(unsigned codeSize, unsigned prologSize
         gcInfoEncoder->SetSizeOfEditAndContinuePreservedArea(preservedAreaSize);
     }
 
+    if (compiler->opts.IsReversePInvoke())
+    {
+        unsigned reversePInvokeFrameVarNumber = compiler->lvaReversePInvokeFrameVar;
+        assert(reversePInvokeFrameVarNumber != BAD_VAR_NUM && reversePInvokeFrameVarNumber < compiler->lvaRefCount);
+        LclVarDsc reversePInvokeFrameVar = compiler->lvaTable[reversePInvokeFrameVarNumber];
```

Comment:
Generally we use `LclVarDsc* var = &compiler->lvaTable[offset]` to avoid copying the entire `LclVarDsc`.